src/Pelagos/Bundle/AppBundle/Resources/public/css/data-discovery.css

```diff
@@ -140,13 +140,15 @@ input.ui-button {
     border-top-right-radius: 4px;
 }
 
+.datasets tr:nth-child(odd) {
```

Comment:
Odd or Even?

Reply:
Coloring the odds so the first row gets colored